src/vm/eetwain.cpp

```diff
@@ -4249,6 +4249,18 @@ bool EECodeManager::EnumGcRefs( PREGDISPLAY     pContext,
 
 #endif // _DEBUG
 
+#ifdef WIN64EXCEPTIONS   // funclets
+    //
+    // If we're in a funclet, we do not want to report the incoming varargs.  This is
```

Comment:
It crashes while [Process the untracked frame variable table](https://github.com/dotnet/coreclr/blob/bcc0a1d528b78f5e05bf7973090f33ef2403e685/src/vm/eetwain.cpp#L4590-L4646). Would funclets need to do something with these untracked variables? I thought they are handled by main function frame.

By the way does untracked frame variable mean ref-type local variables?